x/dist/keeper/proposal_handler.go

```diff
@@ -12,9 +14,9 @@ func HandleCommunityPoolSpendProposal(ctx sdk.Context, k Keeper, p *types.Commun
 		return addrErr
 	}
 
-	// if k.bankKeeper.BlockedAddr(recipient) {
-	// 	return errorsmod.Wrapf(sdkerrors.ErrUnauthorized, "%s is not allowed to receive external funds", p.Recipient)
-	// }
+	if k.bankKeeper.BlockedAddr(recipient) {
```

Comment:
not sure why it was removed before.
it's a check the community spend proposal can't be to blocked addr